app/Http/Controllers/Users/UserController.php

```diff
@@ -45,9 +39,7 @@ public function create()
      */
     public function store(Request $request)
     {
-        $user = User::findOrFail($request->user_id);
-
-        return $this->update($request, $user);
+        //
```

Comment:
Why comment out these functions?

Reply:
These were used by admins in the User display to update permissions

Now that permissions are handled through Discord roles the ability to update permissions through the site was removed

The reason that I've left the functions there, but with no functionality, is I believe User is defined as a REST resource (so should implement CRUD methods)